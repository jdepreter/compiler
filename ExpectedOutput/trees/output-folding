digraph {
	1 [label=C]
	3 [label=scope]
	1 -> 3
	5 [label=definition]
	3 -> 5
	6 [label=int]
	5 -> 6
	7 [label=assignment2]
	5 -> 7
	8 [label=c]
	7 -> 8
	23 [label=24]
	7 -> 23
	73 [label=definition]
	3 -> 73
	74 [label=float]
	73 -> 74
	75 [label=assignment2]
	73 -> 75
	76 [label=f]
	75 -> 76
	102 [label=40]
	75 -> 102
	140 [label=method_call]
	3 -> 140
	141 [label=printf]
	140 -> 141
	150 [label=f]
	140 -> 150
	151 [label="empty arg"]
	140 -> 151
	153 [label=assignment]
	3 -> 153
	154 [label=f]
	153 -> 154
	180 [label=12.999999999999998]
	153 -> 180
	218 [label=definition]
	3 -> 218
	219 [label=int]
	218 -> 219
	220 [label=assignment2]
	218 -> 220
	221 [label=i]
	220 -> 221
	247 [label=12.999999999999998]
	220 -> 247
	285 [label=method_call]
	3 -> 285
	286 [label=printf]
	285 -> 286
	295 [label=f]
	285 -> 295
	296 [label="empty arg"]
	285 -> 296
}
