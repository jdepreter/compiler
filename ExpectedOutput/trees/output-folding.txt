digraph {
	1 [label=C]
	3 [label=scope]
	1 -> 3
	5 [label=definition]
	3 -> 5
	6 [label=int]
	5 -> 6
	7 [label=assignment2]
	5 -> 7
	8 [label=c]
	7 -> 8
	23 [label=24]
	7 -> 23
	73 [label=definition]
	3 -> 73
	74 [label=float]
	73 -> 74
	75 [label=assignment2]
	73 -> 75
	76 [label=f]
	75 -> 76
	102 [label=40]
	75 -> 102
	140 [label=assignment]
	3 -> 140
	141 [label=f]
	140 -> 141
	167 [label=12.999999999999998]
	140 -> 167
	205 [label=definition]
	3 -> 205
	206 [label=int]
	205 -> 206
	207 [label=assignment2]
	205 -> 207
	208 [label=i]
	207 -> 208
	234 [label=12.999999999999998]
	207 -> 234
}
